code/game/objects/items/crayons.dm

```diff
@@ -380,6 +380,9 @@
 	if(edible && (M == user))
 		user << "You take a bite of the [src.name]. Delicious!"
 		var/eaten = use_charges(5)
+		if(!reagents.total_volume) //Prevents divsion by zero
+			check_empty()
```

Comment:
https://github.com/TheCarlSaganExpress/tgstation/blob/b88a4572a4b77f40a6fa1d6283871b580c2e5900/code/game/objects/items/crayons.dm#L132

did you add this code or was it already here? it seems related, because it checks qdeleted (incorrectly) I might add and then doesn't actually do anything other than set the default return